qlf/dashboard/models.py

```diff
@@ -7,8 +7,8 @@ class Exposure(models.Model):
 
     # TODO: make null=False when exposure data is available
 
-    expid = models.CharField(max_length=8, unique=True,
-                                 help_text='Exposure number')
+    exposure_id = models.IntegerField(primary_key=True,
```

Comment:
I see now `exposure_id` is the primary key, which I think is a good improvement and avoid confusing. So my suggestion would be to use always variable names like `exposure_id` and `exposure_ids`.